src/ocamlorg_web/lib/router.ml

```diff
@@ -23,71 +23,71 @@ let media_loader =
       let* store in
       Media.last_modified store (Mirage_kv.Key.v path))
 
-let get_and_head pattern handler = [
-  Dream.get pattern handler;
-  Dream.head pattern handler
-]
+let get_and_head pattern handler =
```

Comment:
We shouldn't need this if the middleware is applied before the router and has the following behaviour:

- If the request is HEAD, change it to GET and pass it to the handler (plus all the logic in the head middleware)
- Else, pass it to the handler